person.rb

```diff
@@ -25,4 +25,8 @@ def can_use_services?
   def of_age?
     @age < 18
   end
+
```

Comment:
- [ ] *@rentals should be initialized. Link to the example about constructors in a previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) 
    @rentals = []